evalplus/codegen.py

```diff
@@ -120,6 +121,7 @@ def run_codegen(
     bs: Optional[int] = None,
     n_samples: int = 1,
     temperature: float = 0.0,
+    num_ctx: Optional[int] = None,

```

Comment:
i hope to simplify the interface as it's getting bloated a bit.
is it possible to drop the `num_ctx` parameter here as it seems this is mostly controlled by the server, not the client side?

Reply:
Well, in Ollama it is controllable from both sides, which makes total sense as you can experience with several runs of benchmarks with a variance of context sizes.